products/sql/ansible.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Ansible::Config
+manifest: !ruby/object:Provider::Ansible::Manifest
+  metadata_version: '1.1'
+  status:
+    - preview
+  supported_by: 'community'
+  requirements:
+    - python >= 2.6
+    - requests >= 2.18.4
+    - google-auth >= 1.3.0
+  version_added: '2.6'
+  author: Google Inc. (@googlecloudplatform)
+# This is where custom code would be defined eventually.
+overrides: !ruby/object:Provider::ResourceOverrides
+  Instance: !ruby/object:Provider::Ansible::ResourceOverride
+    access_api_results: true
+    return_if_object: |
+      # If not found, return nothing.
+      if response.status_code == 404:
+          return None
+
+      # If no content, return nothing.
+      if response.status_code == 204:
+          return None
+
+      # SQL only: return on 403 if not exist
```

Comment:
Add the link to the never-being-fixed-yet-assumed-it-is-a-bad-thing bug here.

Reply:
I don't think we can do that anymore. Those are internal links.